webpack.config.js

```diff
@@ -453,7 +476,46 @@ if (TARGET !== "readme") {
             maxEntrypointSize: prodMode ? 250000 /* the default */ : 10000000
         },
 
-        devtool: prodMode ? "hidden-source-map" : "source-map"
+        devtool: prodMode ? "hidden-source-map" : "source-map",
+
+        devServer: {
+            allowedHosts: "all", // Replace with docker parent and localhost
+            client: {
+                logging: "verbose",
+                overlay: {
+                    errors: true,
+                    warnings: false
+                },
+                progress: true
+            },
+            compress: true,
+            devMiddleware: {
+                index: false,
+                serverSideRender: true,
+                writeToDisk: true // Required for Perl TT
+            },
+            hot: true,
+            port: 9000,
```

Comment:
Locally, I added a row `host: "0.0.0.0",` before the port in order to be able to access the dev server from outside the container I'm running this in.